database/migrations/2018_12_17_124953_create_musics_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateMusicsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('musics', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->int('view_count');
```

Comment:
```suggestion
            $table->integer('view_count');
```
https://laravel.com/docs/5.7/migrations#columns